assets/javascripts/discourse/components/result-types/reltime.js

```diff
@@ -0,0 +1,11 @@
+import Component from "@glimmer/component";
+import { autoUpdatingRelativeAge } from "discourse/lib/formatter";
+import { htmlSafe } from "@ember/template";
+
+export default class Reltime extends Component {
+  get boundDateReplacement() {
+    return htmlSafe(
+      autoUpdatingRelativeAge(new Date(this.args.ctx.value), { title: true })
```

Comment:
I wonder if we should catch 'invalid date' errors here and return "invalid date" or something? Otherwise a bad query could blow up rendering of the component

Reply:
This is already baked in. I think its a moment.js feature :)

```
-- [params]
-- string :string (set it to a bad value)

SELECT
:string as reltime$updated_at,
```
Result: 
<img width="78" alt="Screenshot 2023-03-10 at 11 41 07 AM" src="https://user-images.githubusercontent.com/50783505/224385989-a4b8c22e-7ccb-4a53-80ff-74893a515eb2.png">